README.md

```diff
@@ -21,6 +21,12 @@ npm install
 
 Use `npm run watch` to compile and serve the dist directory in real time.
 
+
+## Add new logo images for sponsors and partners
+
+- Edit image remove background, add whitespace
+- Resize the image to `600 x 400 px`
```

Comment:
600x400px is the size for tera sponsors (at least in my laptop). I rather have a fixed size and modifying in the frontend as its easier to remember and make sure its normalized.